homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
```

Comment:
It looks like merging everything back into one service is going to require some changes to pyharmony, so this'll be on hold until I can another release out with that change in it.